src/PKSim.Core/Services/IndividualModelTask.cs

```diff
@@ -53,6 +64,11 @@ public void CreateModelFor(Individual individual)
          _buildingBlockFinalizer.Finalize(individual);
       }
 
+      public void CreateOrganStructureFor(Individual individual)
```

Comment:
@Yuri05 In fact for ExpressionProfile, I can even make it better by not using neighbourhood. I just need the container structure. The "internal" individual will be more smaller. No parameters, no neighborhood, just contianers